_includes/1.0/overview/filter.html

```diff
@@ -0,0 +1,29 @@
+<div id="filter" class="group">
+    <h2>Filter</h2>
+    <p>
+        API users are able to apply conditions on requests that support the filter feature.
+        Pass allowed parameters mentioned in each section named "Available Filter Options" of endpoints that handle this feature. 
+    </p>
+    <p>
+        Thanks to comparison and conjunction operators filter your allow to retrieve and shape data the way that you need it
+        (e.g. created_at=gte:2020-10-01 AND created_at=lte:2020-10-31).
+    </p>
+
+    <h3 id="comparisonfilter">Comparison operators</h3>
+    <table class="table table-bordered table-striped">
+        <thead>
+        {% include table-header-comparison-ops.html %}
+        </thead>
+        <tbody>
+        <tr>
+            <td>lte:</td>
+            <td><= (less than or equal)</td>
+        </tr>
+        <tr>
+            <td>gte:</td>
+            <td>>= (greater than or equal)</td>
```

Comment:
I guess we're missing operators here, like 'AND' and maybe others. Please test them on some end points, and if they are working properly then add them to the doc.